keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/diff/DualNumber.java

```diff
@@ -32,6 +31,70 @@ public static DualNumber ifThenElse(BooleanTensor predicate, DualNumber thn, Dua
         }
     }
 
+    public static DualNumber concat(Map<Vertex, DualNumber> dualNumbers,
```

Comment:
Looks good for now. We should add a task in the backlog to revisit this, but we should fix this now.